### PR TITLE
Add RoleBinding To New PR Projects

### PR DESCRIPTION
**Pull request by sherl0cks**
@oybed can you take a peek at this one? want to make sure the policy stuff is OK with you. note there is an update to the internal gitlab MR as well.

### CODE REVIEW

File: openshift-templates/rolebinding-group/template.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+metadata:
+  name: jenkins-pipeline-no-ocp-triggers
+objects:
+- apiVersion: v1
+  kind: RoleBinding
+  metadata:
+    name: "${ROLE}"
+  roleRef:
+    name: "${ROLE}"                               
+  groupNames:
```
Comment (oybed) [inline openshift-templates/rolebinding-group/template.yaml]: @sherl0cks did you find that the `subjects` section wasn't needed when adding the `groupNames` section?

Comment (sherl0cks) [reply openshift-templates/rolebinding-group/template.yaml]: it appears that the `subjects` part is generated server side. I'm a bit confused on why it's there to be honest, as it appears to be duplicative info

Comment (sherl0cks) [reply openshift-templates/rolebinding-group/template.yaml]: I did find a typo though... fixed and rerunning CI

**Comment by sherl0cks** [comment]
@oybed is this good to merge now?

File: openshift-templates/rolebinding-group/template.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  name: jenkins-pipeline-no-ocp-triggers
```
Comment (oybed) [inline openshift-templates/rolebinding-group/template.yaml]: Since this is a "generic" template, probably should use `${ROLE}` here as well?

Comment (sherl0cks) [reply openshift-templates/rolebinding-group/template.yaml]: oh right, copy / paste error

**Comment by sherl0cks** [comment]
@oybed should be all tidied up now.

**Pull request merged**